features/events/next_scrum_for_new_user.feature

```diff
@@ -0,0 +1,50 @@
+@vcr
```

Comment:
I'm concerned this file is a heavyweight replication of `next_scrum.feature` which just checks a different text when a user is not logged in - would a view spec be more sensible or sufficient ...?

or a scenario outline?  Or just some tweaking on whether all the set up is needed in the cucumber background?